src/Datasource/RuleInvoker.php

```diff
@@ -136,6 +136,7 @@ public function __invoke(EntityInterface $entity, array $scope): bool
             $entity->setInvalidField($errorField, $invalidValue);
         }
 
+        /** @phpstan-ignore-next-line */
```

Comment:
Why did this change with a psalm version bump?

Reply:
The error is caused by a new phpstan patch release, not pslam. I just chose to update psalm too while I was at it.